core-contracts/FeeDistribution/src/main/java/finance/omm/score/fee/distribution/AbstractFeeDistribution.java

```diff
@@ -51,6 +53,9 @@ protected void distributeFee(BigInteger amount) {
                 accumulatedFee.set(receiver, feeAccumulatedAfterClaim.add(amountToDistribute));
             }
         }
+        if (remaining.compareTo(BigInteger.ZERO) > 0){
```

Comment:
IMO, BigInteger.signum() looks better than this.
Is there any reason why BigInteger.compareTo() is used?

```suggestion
        if (remaining.signum() > 0){
```

Reply:
We always use BigInteger.compareTo() in our code for consistency. While we could use signum(), we stick with compareTo() to keep things uniform. 
But as curiosity is there are any discernible differences in terms of performance or step cost between the two methods.